### ISSUE TITLE
branch coverage

### ISSUE THREAD

**Issue by davidszotten** [nameko/nameko#160]
Now that we're no longer on a coverage fork, we should switch on branch coverage (once we get that up to 100%)

**Comment by jessepollak** [comment]
Did this ever happen? If not, is there still interest in switching to branch coverage? If the answer is yes, we can re-tag this and start putting together a plan to make it happen.

**Comment by davidszotten** [comment]
i don't believe it happened. i think it would still be interesting to do, though perhaps not high priority. i'm not sure we need to get to zero open issues, but maybe we could be categorising issues (e.g. this could be "some-day" or whatever)

as for tactics, an initial step might be to add some travis targets that run with branch coverage and allow-failure (or without the limit) just to get a sense of how we are doing, or at least a local run to see how far off this we are atm

**Comment by jessepollak** [comment]
@davidszotten agreed re: 0 issues and categorization. unfortunately, I need write access before I can make labels :(

**Comment by davidszotten** [comment]
sure, though as per my note on the mailing list, while we work on getting you access, maybe you could have a thinkg and propose a set of tags and/or other processes to the mailing list

**Comment by mattbennett** [comment]
Branch coverage is at 99% 😮 

I've added an allowed failure to travis in https://github.com/onefinestay/nameko/pull/372
